ProjectCars/CarCRUD.cs

```diff
@@ -40,24 +38,87 @@ public static List<Car> GetCars()
 
         public Car Create(Car car)
         {
-            
-            return null;
+            var cars = getCarsJsonFromFile();
+            var carsList = DeserealizeJsonFile(cars);
+            carsList.Add(car);
+            SerializeJsonFile(carsList);
+            //string carStr = car.ToString();
+            //File.AppendAllText(_path, carStr);
+            return car;
         }
 
-        public Car Get(int Id)
+        public Car Get(string Id)
         {
+            var cars = getCarsJsonFromFile();
+            var carsList = DeserealizeJsonFile(cars);
+            foreach (var car in carsList)
+            {
+                if (car.Id == Id)
+                {
+                    return car;
+                }
+            }
             return null;
         }
 
-        public Car Update(Car car)
+        public Car Update(Car carUpd)
         {
-            return null;
+            var actualized = false;
```

Comment:
I want to challenge you, consider this an exercise: try to achieve the update functionality without using this flag.